pkg/guestagent/procnettcp/procnettcp.go

```diff
@@ -15,14 +15,17 @@ type Kind = string
 const (
 	TCP  Kind = "tcp"
 	TCP6 Kind = "tcp6"
-	// TODO: "udp", "udp6", "udplite", "udplite6".
+	UDP  Kind = "udp"
+	// TODO: "udplite", "udplite6".
```

Comment:
what about udp6?

Reply:
My bad, missed it. Will enable for udp6 as well.

For udplite, updlite6 we can enable in a separate PR. It should work as is mostly.

Reply:
Done.